.gitlab/e2e.yml

```diff
@@ -324,6 +324,23 @@ new-e2e-orchestrator-main:
 
   allow_failure: true #TODO: Remove when https://github.com/DataDog/datadog-agent/pull/22113 is merged
 
+new-e2e-test-infra-definition-dev:
```

Comment:
Should we have other `new-e2e` depend on these to prevent their execution if these are failing?

Reply:
Yes, will add a new stage for this

Reply:
I am wondering if we should limit runs of this to when we update the test-infra version, and have it before the installer tests, maybe based on latest stable. Ideally we should run this also based on a trigger from `test-infra` to validate changes on that side.

Reply:
Updated, moved this to a new stage to run only on branches with changes on new-e2e framework code and dependencies